Core/Frameworks/Baikal/Resources/Templates/Main.html

```diff
@@ -0,0 +1,14 @@
+{% autoescape false %}
+<header class="jumbotron subhead" id="overview" style="padding-bottom:5em;">
+	<h1><img style="height:150px; float:left;" src="res/core/Baikal/Images/logo-baikal.png" /> Baïkal
+	<p class="lead">Lightweight CalDAV+CardDAV server.</p></h1>
+</header>
+
+{{ message }}
+
+<section id="about">
```

Comment:
The line break looks a bit strange on small screens. I would prefer the text `Baïkal is running alright` to start below the icon. It can be fixed by adding `<br />` above this line but I guess it can also be done with css.

![image](https://user-images.githubusercontent.com/5811634/56530197-47553180-6552-11e9-9acd-c2c6fc5f5925.png)

Reply:
fixed in da9f981ae99e258922fe07f1483ba99902c54e7d